openpype/settings/defaults/project_settings/deadline.json

```diff
@@ -95,6 +68,33 @@
             "group": "",
             "department": "",
             "multiprocess": true
+        },
+        "ProcessSubmittedJobOnFarm": {
+            "enabled": true,
+            "deadline_department": "",
+            "deadline_pool": "",
+            "deadline_group": "",
+            "deadline_chunk_size": 1,
+            "deadline_priority": 50,
+            "publishing_script": "",
+            "skip_integration_repre_list": [],
+            "aov_filter": {
+                "maya": [
+                    ".*(?:[\\._-])*([Bb]eauty)(?:[\\.|_])*.*"
```

Comment:
The extra * after both of the non capture groups `(?:[\\._-])` & `(?:[\\.|_])` sounds off. We allow zero, one or more occurrences of it? If so, why do we even try and match it if it can also not be there at all?
`(?:` marks a non capture group. So the group it captures can be ignored.
Then after the non capture group we allow anything any amount of times `.*`?

If we don't care about `(?:[\\.|_])*` this particular match. Couldn't we just make this?
```
.*([Bb]eauty).*
```

Reply:
True that simpler regex works too, from a quick test. Keep in mind also that this is technically an example default, that often needs adjusting in the studio, but I agree, let's keep it simpler.